### ISSUE TITLE
failed: RollingUpgradeSpec

### ISSUE THREAD

**Issue by patriknw** [akka/akka#29705]
I think this is failing frequently at the moment

https://travis-ci.com/github/akka/akka-upgrade-testing/builds/184044641

```
[info] RollingUpgradeSpec:
1842[info] Rolling upgrade
1843[info] - should build images
1844[info] - should deploy an initial cluster then upgrade through all the versions
1845[info] - should upgrade to version 2.6.5
1846[info] - should upgrade to version 2.6.9+11-51e32412 *** FAILED ***
1847[info]   akka-upgrade-testing-blue-5f564cf7c8-n7nhx    1/1     Running             0          2m20s
1848[info]   akka-upgrade-testing-blue-5f564cf7c8-nj6tf    1/1     Running             0          3m4s
1849[info]   akka-upgrade-testing-blue-5f564cf7c8-z5w6q    1/1     Running             0          55s
1850[info]   akka-upgrade-testing-green-688d9c79b5-h8krr   0/1     ContainerCreating   0          0s The code passed to eventually never returned normally. Attempted 44 times over 3.0624731147 minutes. Last failure message: false did not equal true. (RollingUpgradeSpec.scala:173)
```

**Comment by octonato** [comment]
Retriggered the build and it's now green. I will close this one.

**Comment by octonato** [comment]
Fixed in #29692
